README.md

```diff
@@ -11,13 +11,13 @@ The easiest way to do this is to initialize a "custom" project with `expo-cli`.
 ### Install the package
 
 ```bash
-npm install @unimodules/core
+npm install react-native-unimodules
 ```
 
 ### Configure iOS
 
-- Go back to the `ios` directory and open your Podfile, make your Podfile look [like this one](https://gist.github.com/brentvatne/6a1dcb32f6ca3d478eed4c7dc8fbdd24).
-  - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `use_unimodules!(node_modules_path: '../../../node_modules')`
+- Go back to the `ios` directory and open your Podfile, make your Podfile look [like this one](https://gist.github.com/sjchmiela/6c079f2173938a9a61a7c6f053c45000).
+  - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `use_unimodules!(modules_paths: '../../../node_modules')`
```

Comment:
```suggestion
  - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `use_unimodules!(modules_paths: ['./path/to/node_modules'])`
```